src/app/domains/Form/routes/FormsAll/FormsAll.js

```diff
@@ -17,45 +20,45 @@ import {
   FilterOutlined,
   PlusOutlined
 } from '@ant-design/icons'
+import PropTypes from 'prop-types'
+import { Link } from 'react-router-dom'
+import { firestore } from 'app/services'
 import { useHistory } from 'react-router'
 import { globalStyles } from 'app/styles'
 import { styles } from './FormsAll.styles'
-import { Link } from 'react-router-dom'
-import { FormSimpleView } from 'domains/Form/components'
-// import { useTranslation } from 'react-i18next'
+import { FormSimpleForm, FormSimpleView } from 'domains/Form/components'
+import { useCollectionData } from 'react-firebase-hooks/firestore'
+import { setData } from 'app/services/Firestore'
+import COLLECTIONS from 'app/constants/collection'
+import { Spinner } from 'components'
 
 const { Title, Text } = Typography
-
 const mockRoutes = [
   { path: '/forms', page: 'Forms' },
   { path: '/images', page: 'Images' },
   { path: '/videos', page: 'Videos' }
 ]
-
+const mockList = [0, 1, 2, 8, 9, 10, 11, 6, 7, 8, 9, 10, 6, 7, 8, 9, 10]
 function FormsAll(props) {
   //const { WRITE_PROPS_HERE } = props
   // const { ADDITIONAL_DESTRUCTURING_HERE } = user
 
   // [ADDITIONAL HOOKS]
-  // const { t } = useTranslation('translation')
-  // const { currentLanguage } = t
   const history = useHistory()
+  const [data] = useCollectionData(firestore.collection('forms'))
```

Comment:
Please, use const variable for the collection name (import it from appropriate place)

Reply:
Done